README.md

```diff
@@ -44,6 +44,14 @@ or
 
 (Optional) Disable Django debug toolbar and contracts for faster tests: https://github.com/edx/edx-platform/wiki/Developing-on-the-edX-Developer-Stack#making-the-local-servers-run-faster
 
+Install webdriver:
+	
+	node_modules/protractor/bin/webdriver-manager update --standalone
+
+Edit **protractor.conf.js** and change the version of seleniumServerJar to:
```

Comment:
Since you already updated the protractor.conf.js, there is no need for manually updating this file. So could you remove this step? Thanks.